db/batchlog_manager.hh

```diff
@@ -69,6 +69,8 @@ private:
     seastar::abort_source _stop;
     future<> _loop_done;
 
+    gc_clock::time_point _last_replay;
+
```

Comment:
Why gc_clock? It's not related to gc. Better to use lowres_clock.

Reply:
It is related to gc because we use the last replay time to decide the repair_time which is used to do gc.